packages/core/test/middleware.js

```diff
@@ -95,22 +95,22 @@ test('middleware options', assert => {
   assert.end()
 })
 
-test("middleware's custom emitter", assert => {
+test('lift a channel with a custom emitter', assert => {
   const actual = []
 
   function* saga() {
     yield takeEvery('*', ac => actual.push(ac.type))
   }
 
-  const middleware = sagaMiddleware({
-    emitter: emit => action => {
-      if (action.type === 'batch') {
-        action.batch.forEach(emit)
-        return
-      }
-      emit(action)
-    },
-  })
+  const emitter = emit => action => {
+    if (action.type === 'batch') {
+      action.batch.forEach(emit)
+      return
+    }
+    emit(action)
+  }
+  const channel = stdChannel().lift(emitter)
```

Comment:
This shows how a customized emitter could be applied.

Reply:
this looks quite nice